File: backend/internal/models/output_volume.go
package models

type OutputVolume struct {
	Id       string `json:"id" db:"id"`
	ForStock bool   `json:"forStock" db:"for_stock"`
	Date     int64  `json:"date" db:"date" binding:"required"`
	Product  string `json:"product" db:"product" binding:"required"`
	Count    string `json:"count" db:"count"`
	Money    string `json:"money" db:"money"`
}

File: frontend/src/types/outputVolume.ts
export interface IOutputVolume {
	id: string | null
	date?: string
	forStock?: boolean
	product: string | null
	count: number | null
	money: number | null
}

export interface IOutputVolumeDTO {
	id: string
	forStock: boolean
	date: number
	product: string
	count: string
	money: string
}

export interface IOutput {
	id: string
	date: string
	forStock: boolean
	product: string
	count: number
	money: number
	planQuantity: number
	planMoney: number
}
